src/hooks/__stories__/useClickOutside.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useRef, useState } from "react";
```

Comment:
Hi @LastExileUA why did you add this file, we moved the hooks stories to a different location.

can you please go over these files and remove what is not necessary 

Thank you